### PR TITLE
MCKIN-29094(<McKA> UI - Loader icon is replaced with a new "Load more" button while applying filters on the posts), MCKIN-29096(<McKA> Main discussions/Inline discussions - On creating a new post, the string “Required“ is added on integration.)

### PR DESCRIPTION
**Pull request by wasifarbisoft**
MCKIN-29094(<McKA> UI - Loader icon is replaced with a new "Load more" button while applying filters on the posts)
MCKIN-29096(<McKA> Main discussions/Inline discussions - On creating a new post, the string “Required“ is added on integration.)

This pull Request also adds the translations that were missing due to rebase activity. for languages(French,  Spanish, Portuguese)

### CODE REVIEW

File: conf/locale/es_419/LC_MESSAGES/djangojs.po [hunk 1]
```diff
@@ -9787,10 +9829,13 @@ msgstr ""
 
 #: openedx/features/learner_profile/static/learner_profile/templates/share_modal.underscore
 msgid ""
-"{download_link_start}Download this image (right-click or option-click, save "
-"as){link_end} and then {upload_link_start}upload{link_end} it to your "
-"backpack."
+"%(download_link_start)sDownload this image (right-click or option-click, "
+"save as)%(link_end)s and then %(upload_link_start)supload%(link_end)s it to "
+"your backpack.</li>"
```
Comment (musmanmalik) [inline conf/locale/es_419/LC_MESSAGES/djangojs.po:9834]: Is this closing tag a part of string? Please verify this.

File: conf/locale/es_419/LC_MESSAGES/djangojs.po [hunk 2]
```diff
@@ -2903,9 +2907,11 @@ msgstr "¿Deseas abandonar este equipo?"
 
 #: lms/djangoapps/teams/static/teams/js/views/team_profile.js
 msgid ""
-"If you leave, you can no longer post in this team's discussions.Your place "
+"If you leave, you can no longer post in this team's discussions. Your place "
```
Comment (musmanmalik) [inline conf/locale/es_419/LC_MESSAGES/djangojs.po:2910]: Here an addition of space after period, does it exists in source string ? Please verify is this working for you.

File: conf/locale/es_419/LC_MESSAGES/djangojs.po (hunk 1 above)
Comment (wasifarbisoft) [reply conf/locale/es_419/LC_MESSAGES/djangojs.po]: Fixed! Thanks!

File: conf/locale/es_419/LC_MESSAGES/djangojs.po (hunk 2 above)
Comment (wasifarbisoft) [reply conf/locale/es_419/LC_MESSAGES/djangojs.po]: Fixed and verified1

**Comment by wasifarbisoft** [comment]
@musmanmalik 
I have Updated the translations!

**Review by musmanmalik — APPROVED** [review]
LGTM, let it reviewed by someone from team edx before merging.

**Comment by hasnain-naveed** [comment]
@wasifarbisoft I have not a particular issue with this PR but I am curious you are adding few new strings in few languages why not on all the languages? Please give me some context.

**Comment by wasifarbisoft** [comment]
@hasnain-naveed ,
While compiling the translation we found that there are few string whose translations are missing. The translations are there in the original development branch but were missed due to the rebase activity. I have fixed those for the languages French, Spanish, and Portuguese.

But for the rest of the languages there is discovery ticket being created to investigate missing translation and will be fixed afterwards.
Discovery ticket is
https://edx-wiki.atlassian.net/browse/MCKIN-29336

**Review by hasnain-naveed — APPROVED** [review]

**Pull request merged**